main.go

```diff
@@ -64,6 +64,7 @@ func main() {
 	flagSet.String("bitbucket-repository", "", "restrict logins to user with access to this repository")
 	flagSet.String("github-org", "", "restrict logins to members of this organisation")
 	flagSet.String("github-team", "", "restrict logins to members of this team")
+	flagSet.String("github-user", "", "restrict logins to these users")
```

Comment:
We have a number of options that allow the flag to be specified multiple times, eg `whitelist-domain`, would it be possible to follow the same pattern?

Reply:
fixed by using `flagSet.StringSlice`.